http-client/src/main/java/io/airlift/http/client/HttpClientModule.java

```diff
@@ -124,6 +127,8 @@ public void setInjector(Injector injector)
         @Override
         public HttpClient get()
         {
+            KerberosConfig kerberosConfig = injector.getInstance(Key.get(KerberosConfig.class));
```

Comment:
Is the `Key.get` necessary?  Does `injector.getInstance(KerberosConfig.class)` work?